consumer/src/app/(purchase)/carrinho/components/card-produto-cart.tsx

```diff
@@ -9,23 +9,36 @@ export default function CardProdutoCart({
   product: ProductCart;
   exclude: boolean;
 }) {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const toggleDescription = (offer: any) => {
+    offer.expandDescription = !offer.expandDescription;
+    setIsExpanded(!isExpanded);
+  };
 
   const mapQuantity = {
-    "UNIT": 1,
-    "WEIGHT": 500
+    UNIT: 1,
+    WEIGHT: 500,
   };
-  
+
+  const mapPriceText = {
+    UNIT: "unid.",
+    WEIGHT: "500g",
+  };
+
   const mapTextQuantity = {
```

Comment:
Existe um conversor na pasta utils do shared, utiliza lá pra não ter código duplicado.
Inclusive tu pode ver um exemplo da utilização em: cdd/src/app/(protected)/(footered)/ofertas/[box_id]/components/page.tsx

Reply:
@ilucaslima No mini app é utilizadao a informação 500g e unid e outros cenários é utilizado kg ou unid. 
Nesse mapamento é utilizado apenas un. e kg.

Reply:
@Giovannicm Perfeito, lá foi criado os cenários no qual estamos utilizando, como se baseia em um conversor, você cria os seus cenários também por que no dia que eu precisar usar um cenário igual o seu, eu só faço puxar de lá e você consegue reutilizar quando necessário.
Como é a mesma variável 'WEIGHT', você cria mais uma pra se referir sempre que for WEIGHT (em gramas).